<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QRadioButton" name="radioButtonFirstClass">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>90</y>
     <width>241</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>First Class $150</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>40</y>
     <width>251</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Choose the flight type</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelFare">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>230</y>
     <width>251</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButtonBussinessClass">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>130</y>
     <width>241</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Bussiness Class $125</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButtonEconomyClass">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>170</y>
     <width>191</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Ecnomics Class $100</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
